instrumentation/graphql-java/graphql-java-20.0/library/src/main/java/io/opentelemetry/instrumentation/graphql/v20_0/GraphQLTelemetryBuilder.java

```diff
@@ -27,11 +31,31 @@ public GraphQLTelemetryBuilder setSanitizeQuery(boolean sanitizeQuery) {
     return this;
   }
 
+  /** Sets whether spans are created for GraphQL Data Fetchers. Default is {@code false}. */
+  @CanIgnoreReturnValue
+  public GraphQLTelemetryBuilder createSpansForDataFetchers(boolean createSpansForDataFetchers) {
```

Comment:
Looking at the existing TelemetryBuilder classes I think `setDataFetcherInstrumentationEnabled(boolean enabled)` would better align with existing code.
Also typically there is a configuration flag in the javaagent instrumentation to enable corresponding functionality.

Reply:
I was thinking of updating the Java agent after this PR with the new options but am happy to make the changes now if required.